liquibase-core/src/main/java/liquibase/command/CommandArgumentDefinition.java

```diff
@@ -146,6 +150,28 @@ public String toString() {
         return returnString;
     }
 
+    /**
+     * Interactively prompt in the CLI for a user to supply a value for this command argument.
+     * @param previouslyPromptedValues the values already provided by the user for other arguments in this interactive
+     *                                 prompting session
+     * @return the value provided by the user, or null if the shouldPrompt method of {@link InteractivePromptableCustomizationWrapper}
+     * signifies that this prompt should not occur
+     */
+    public DataType interactivelyPrompt(Map<String, Object> previouslyPromptedValues) {
+        List<DynamicRuleParameter> collect = previouslyPromptedValues.entrySet().stream().map(v -> new DynamicRuleParameter(v.getKey(), v.getValue())).collect(Collectors.toList());
```

Comment:
This seems to be hard-coding in the DynamicRuleParameter here in generic/API code? This layer of the code shouldn't have rule-specific references in it?

Reply:
Absolutely not. I intend/intended (perhaps forgot) to refactor this out. Again, the issue here is that this came over from the prompting from `checks customize` and I was trying to not make breaking changes.